packages/xsnap/lib/confined-object-inspect.js

```diff
@@ -0,0 +1,15 @@
+import objectInspectSources from '../dist/src-object-inspect.js';
+
+// Ensure the object inspector is confined.
+const c = new Compartment();
+harden(c.globalThis);
+
+const src = objectInspectSources.replace(
+  /(^|\s)(export\s+default)(\s+)/g,
+  '$1/* $2 */$3',
+);
```

Comment:
OMG. Definitely worth a comment to explain why.

Reply:
Defineitely!